Runtime_Support/ruby_support.md

```diff
@@ -0,0 +1,20 @@
+# .NET Core on App Service
+
+## .NET Core Update Policy
+
+App Service upgrades the underlying .NET Core runtime and SDK of your application as part of the regular platform updates. As a result of this update process, your application will be automatically updated to the latest patch version available in the platform for the configured minor version of your app.
+
+### End of Life
+
+Once a version of .NET Core has reached it's end of life (EOL) it will no longer be available from Runtime Stack selection dropdown.
+
+Existing applications configured to target a runtime version that has reached EOL should not be affected.
+
+## Support Timeline
+
+| Version  | Support Status  |   End of Support  |   OS Support    |
+|----------| --------------- | ----------------- |---------------- |
+| Ruby 2.6 | End of Life     | TBA               | Linux |
```

Comment:
is the EOL refering to *our* support for ruby, or the community support for ruby
https://www.ruby-lang.org/en/downloads/branches/

Reply:
fixing this with #27